README.md

```diff
@@ -36,56 +47,83 @@ RP_PRIVATE_HEX_KEY=850e54b92c6291a1ff7b8c3ef30e032571ed77c9e5c78b1cd6ee5fec4fea9
 AUTH_REQUEST_EXPIRES_AFTER_SEC=300
 MOCK_AUTH_RESPONSE=false
 ```
+
 (Exception for the IP address this is a valid configuration to test with.)
 
-#### Build & start
-clone the repository:
+From the root directory
+
 ```bash
-git clone git@github.com:Sphereon/onto-web-demo.git
+cd <workspace>/onto-web-demo
+yarn global add concurrently
+yarn global add ts-node
+yarn install-all
+yarn build-types
+yarn start
 ```
-From the root directory
-- yarn global add concurrently
-- yarn global add ts-node
-- yarn install-all
-- yarn build-types
-- yarn start
 
-The server will start on port 5001, the client will start & open a browser on http://localhost:3000/
+The server will start on port `5001`, the client will start & open a browser on `http://localhost:3000/`
 
 #### Docker
+
 From the root folder run:
+
 ```bash
+cd <workspace>/onto-web-demo
 docker build -t onto-web-demo .
```

Comment:
During docker build -- Getting the same error on build-types

 => ERROR [6/6] RUN yarn build-types                                                                                                                            2.6s
------
 > [6/6] RUN yarn build-types:
#9 0.768 yarn run v1.22.5
#9 0.795 $ cd onto-demo-shared-types && yarn run build
#9 0.929 $ tsc -b
#9 2.465 src/auth-model.ts(21,10): error TS2339: Property 'userName' does not exist on type 'AuthResponse'.